priv/cabbage/apps/itest/lib/client.ex

```diff
@@ -157,8 +166,32 @@ defmodule Itest.Client do
     |> ExRLP.encode()
   end
 
-  def wait_until_tx_sync_to_watcher(tx_id) do
-    do_wait_until_tx_sync_to_watcher(tx_id, @default_retry_attempts)
+  def wait_until_tx_sync_to_watcher(tx_id, retry_attempts \\ nil) do
```

Comment:
is this being used?

Reply:
if yes, where, I can't find any reference to it. (specifically wait_until_tx_sync_to_watcher/2)